learningresources/views.py

```diff
@@ -33,6 +37,12 @@ def welcome(request):
 
 
 @login_required
+@permission_required_or_403(
```

Comment:
In the review meeting, we discussed returning a 404 instead of a 403, for better security. django-guardian doesn't appear to have a comparable permission_required_or_404 decorator. I've opened a new issue for resolving this so we don't block this PR.